apps/state/lib/state/routes_patterns_at_stop.ex

```diff
@@ -48,11 +48,12 @@ defmodule State.RoutesPatternsAtStop do
 
   def routes_by_stops_and_direction(stop_ids, opts \\ []) when is_list(stop_ids) do
     direction_id = Keyword.get(opts, :direction_id, :_)
+    ignore? = if Keyword.get(opts, :ignore?, true) == false, do: :_, else: true
```

Comment:
Why the `== false` in addition to the default value? In case someone passes in a non-boolean value for `ignore?`?

Reply:
The default behavior is the same as passing `ignore?: true` as an option. The new feature is to support `ignore: false` which includes both sets of stop IDs.

Reply:
I just meant the default value in `Keyword.get/3`.

e.g. `ignore? = if Keyword.get(opts, :ignore?, true), do: true, else: :_`

Reply:
Ah, gotcha: fixed.